ReportingServicesTools/Functions/CatalogItems/Set-RsSubscription.ps1

```diff
@@ -0,0 +1,148 @@
+﻿# Copyright (c) 2016 Microsoft Corporation. All Rights Reserved.
+# Licensed under the MIT License (MIT)
+
+function Set-RsSubscription
+{
+    <#
+        .SYNOPSIS
+            This script set a new reporting subscription.
+        
+        .DESCRIPTION
+            This script set a new reporting subscription based on the info retrieved using Get-RsSubscription.
+            You can choose a specific report or pass a folder. When using a folder, the report must have the same name.
+            NOTE: A new subscriptionID will be generated.
+                    
+        .PARAMETER Path
+            Specify the path to the destination report. Can't be used with -RsFolder parameter.
+        
+        .PARAMETER RsFolder
+            Specify the folder where the destination reports exists. Can't be used with -Path parameter.
+
+        .PARAMETER Subscription
+            A object with all subscritpion configurations. The default output of Get-RsSubscription. You must use pieing it to this command.
+        
+        .PARAMETER ReportServerUri
+            Specify the Report Server URL to your SQL Server Reporting Services Instance.
+            Use the "Connect-RsReportServer" function to set/update a default value.
+        
+        .PARAMETER Credential
+            Specify the password to use when connecting to your SQL Server Reporting Services Instance.
+            Use the "Connect-RsReportServer" function to set/update a default value.
+        
+        .PARAMETER Proxy
+            Report server proxy to use.
+            Use "New-RsWebServiceProxy" to generate a proxy object for reuse.
+            Useful when repeatedly having to connect to multiple different Report Server.
+        
+        .EXAMPLE
+            Get-RsSubscription -ReportServerUri 'http://localhost/ReportServer_sql14' -Path '/path/to/my/subscription' | Set-RsSubscription -ReportServerUri 'http://remote-machine:8080/reportserver_sql16' -Path '/path/to/new report'
```

Comment:
I would change the sample to say /my/oldReport , having subscription in the name might make you think you are using a subscription path